nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -325,7 +324,7 @@ class VanillaDataManagerConfig(DataManagerConfig):
     camera_optimizer: CameraOptimizerConfig = CameraOptimizerConfig()
     """Specifies the camera pose optimizer used during training. Helpful if poses are noisy, such as for data from
     Record3D."""
-    collate_fn: Callable[[Any], Any] = staticmethod(nerfstudio_collate)
+    collate_fn: Callable[[Any], Any] = nerfstudio_collate
```

Comment:
Can you please use `field(default=nerfstudio_collate)`

Reply:
Do you mind explaining? Is this a static vs non-static method thing?

Reply:
Sorry, I think your code is fine.